src/components/token/TokenAlerts.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenAlerts = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken(props.token);
+  }, [props.token]);
+
+  const bannedAlert = () => {
+    if (!token.banned) {
+      return null;
+    }
+
+    return (
+      <div className="alert alert-danger backup-alert" role="alert">
+        <i className="fa fa-exclamation-triangle mr-2" title="Token Verified"></i>
+        This token was banned and should not be trusted. <a href="http://google.com" >Learn more.</a>
```

Comment:
Should we move this message to the Metadata? The only con I can see is that it won't be easy to have it translated in the future.

Reply:
And we need to repeat this message in all banned tokens metadata, right? If we decide to change a word, then we would need to change all metadatas, or am I missing something?

Reply:
We can have a key related to a i18n message instead of a message. However i think it's even better to have a default message and we can:
1. Override this message if the metadata have some different message
2. Have an extra message to have some additional information about the ban

For translating, the message field can be in the following format:
```
message: {
  "default": "Default message, probably in english",
  "pt_BR": "Mensagem padrão, provavelmente em inglês"
}
```

Reply:
I understand that we might need to repeat the message in all banned tokens. On the other hand, we can have special cases where we want to place a different message.

But I agree that we should let the wallets and the public explorer decide which message to show. So, we need to put a code in the metadata (e.g., "REGULAR_BAN"). In the future, we can create other codes as needed.

Reply:
In regards to translation, we should use PO files so we can use external translators. We have already built all tools to easily do that in `hathor-wallet` and `hathor-wallet-mobile`. We just have to port them to this repository.